dir.props

```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```

Comment:
https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L1217

Just set that property to disable installing this package locally as it isn't needed for you guys.